src/countOccurrences.js

```diff
@@ -18,7 +18,13 @@
  *
  */
 function countOccurrences(phrase, letter) {
-  // write code here
+  let counter = 0;
+  for (let i = 0; i < phrase.length; i++) {
+    if (phrase[i].toUpperCase() === letter.toUpperCase()) {
+      counter++;
+    }
+  }
```

Comment:
new line before return
```suggestion
  }
  
```